packages/asc-core/src/components/Icon/Icon.ts

```diff
@@ -6,17 +6,20 @@ import getThemeColor from '../../utils/getThemeColor'
 type Props = {
   color?: Theme.TypeLevel
   iconUrl?: string
+  size?: number
```

Comment:
Why not give `size` a default value of `20`?

Reply:
👍